docs/en/stack/security/get-started-tls.asciidoc

```diff
@@ -0,0 +1,429 @@
+running.
+
+. Launch the {kib} web interface by pointing your browser to port 5601. For 
+example, http://127.0.0.1:5601[http://127.0.0.1:5601].
+
+. <<get-started-license,Install a trial license>>.
+
+. <<get-started-enable-security,Enable security in {es}>>.
+
+. <<get-started-built-in-users,Create passwords for built-in users>>.
++
+--
+NOTE: This section of the tutorial does not use the built-in `kibana`, 
+`logstash_system`, and `beats_system` users. To learn more about those users, 
+see <<built-in-users>>.
+
+--
+
+. <<get-started-kibana-user,Add the built-in user to {kib}>>.
+
+. To verify that security is enabled, run the cat health API. For example:
++
+--
+[source,sh]
+----------------------------------
+curl -s -u elastic:<password> -X GET "http://localhost:9200/_cat/health?v"
+----------------------------------
+
+Supply a valid user ID (such as `elastic`) and password, such as the one you 
+created in <<get-started-users>>. 
+
+--
+
+[role="xpack"]
+[[get-started-certificates]]
+=== Generate certificates
+
+In a secured cluster, {es} nodes, {kib} instances, {ls} hosts, and Beats use 
+certificates to identify themselves when accepting incoming connections and 
+connecting to other services. 
+
+The cluster must validate the authenticity of these certificates. The 
+recommended approach is to trust a specific certificate authority (CA). Thus 
+when nodes are added to your cluster they just need to use a certificate signed 
+by the same CA and the node is automatically allowed to join the cluster. 
+
+. Pick a verification mode for the certificates. By default, {es} verifies that 
+the certificate is signed by a trusted CA and also verifies that the server's 
+hostname or IP address matches the names specified within the certificate. If 
+you don't want the hostname verification to occur, add the following setting to 
+the `ES_PATH_CONF/elasticsearch.yml` file:
++
+--
+[source,yaml]
+----
+xpack.ssl.verification_mode: certificate
+----
+
+In this tutorial, we will use the default value of `full`.
+
+For more information about this setting, see 
+{ref}/security-settings.html[Security settings in {es}]. 
+--
+
+. Generate a certificate authority. For example, use the `elasticsearch-certutil` 
+tool by running the following command from the {es} directory:
++
+--
+["source","sh",subs="attributes,callouts"]
+----------------------------------------------------------------------
+./bin/elasticsearch-certutil ca
+----------------------------------------------------------------------
+// NOTCONSOLE
+
+By default, the command outputs a single file with the name 
+`elastic-stack-ca.p12`. This file is a PKCS#12 keystore that contains the public
+certificate for your CA and the private key that is used to sign the node 
+certificates.
+
+The `elasticsearch-certutil` command also prompts you for a password to protect 
+the file and key. If you plan to add more nodes to your cluster in the future, 
```

Comment:
users should always retain this file and store it securely. It holds the key to their cluster